src/app/footer/footer.component.html

```diff
@@ -0,0 +1,3 @@
+<footer>
+    <marquee>sugar-rush - 06/2023 - l'information décryptée sur le sucre ...</marquee>

```

Comment:
The marquee tag is depreciated, can you use another one please.
Even if the scroll effect is not here anymore